### DISCUSSION TITLE
How to solve: AssertionError: y_train_all must be a numpy array or a pandas series.

### DISCUSSION THREAD

**Discussion by prateekgml** [microsoft/FLAML#578] [Q&A] [answered]
Hi,

While working on [https://www.kaggle.com/competitions/amex-default-prediction/data](url), I am facing the following error in fitting the model:

![image](https://user-images.githubusercontent.com/97145738/172594426-dfc4d34c-08f5-4ce9-a622-c15cc5eeee95.png)

What may be the possible cause of this issue?

**Comment by liususan091219** [comment, marked as answer]
Hi @prateekgml thanks for your question. I can’t access the kaggle url, also do you mind sharing your code for train_x and train_y so we can reproduce the issue? Thanks

**Reply by prateekgml** [reply]
Hi @liususan091219 ,

can you please share your kaggle's username so that I can share my notebook?

**Reply by prateekgml** [reply]
@liususan091219 I am attaching here my notebook-
[amex-predict.txt](https://github.com/microsoft/FLAML/files/8861588/amex-predict.txt)

Please change the extension from .txt to .ipynb.

**Reply by liususan091219** [reply]
Thanks for sharing your code @prateekgml. The reason you are seeing this error is that train_y should be a pandas.Series instead of a pandas.DataFrame.
